src/UIComponents/SaveModel.jsx

```diff
@@ -15,24 +15,40 @@ class SaveModel extends Component {
     saveStatus: PropTypes.string
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showColumnDescriptions: false
+    };
+  }
+
+  toggleColumnDescriptions = () => {
+    this.setState({showColumnDescriptions: !this.state.showColumnDescriptions})
+  };
+
   handleChange = (event, field, isColumn) => {
     this.props.setTrainedModelDetail(field, event.target.value, isColumn);
   };
 
-  getFields = () => {
+  getColumnFields = () => {
     var fields = [];
 
-    fields.push({ id: "name", text: "What will you name the model?" });
-
     for (const columnDescription of this.props.columnDescriptions) {
+      const columnType = columnDescription.id === this.props.labelColumn
+        ? "label" : "feature";
       fields.push({
         id: columnDescription.id,
         isColumn: true,
-        text: "Description for: " + columnDescription.id,
-        answer: columnDescription.description
+        text: "Description for: " + columnDescription.id + ` (${columnType})`,
```

Comment:
It's a bit odd to mix `+` and string interpolation in one line.. maybe the whole thing could be generated using string interpolation.